js/battle.js

```diff
@@ -4129,16 +4139,24 @@ var Battle = (function () {
 					actions += "" + poke.getName() + " transformed!";
 					if (toId(template.species) === 'shaymin') break;
 				} else if (toId(template.species) === 'darmanitanzen') {
```

Comment:
That's a server-side change, yes? I'll work on that in a bit, then.

Speaking of Zen Mode, @Marty-D, what happens if a Zen-Mode Darmanitan Skill Swaps another Zen-Mode Darmanitan in-game? (assuming both are under half health)